signatures.md

```diff
@@ -10,40 +10,37 @@ We use signatures in filecoin to verify *something* was done by *someone*. For e
 - Tickets (Signature of proof - [Mining](mining.md)
 - Block signature (Signature over all data in the block - done by block leader)
 
-# What signatures affect
+## Filecoin components that signatures affect
 
-What uses them
+- [Messages](https://github.com/filecoin-project/specs/blob/master/data-structures.md)
+- [Block validation](https://github.com/filecoin-project/specs/blob/master/mining.md)
+- [Tickets](https://github.com/filecoin-project/specs/blob/master/mining.md)
 
-- Messages [TODO: Link to messages spec]
-- Block validation
-- Tickets which inform leader election in Expected Consensus (EC) [TODO: link to EC spec]
+Note that messages between actors are not signed. This is an unfortunate namespace collision. Messages between actors are always spawned by a message from a user. In this way, signed messages are containers for messages between actors.
 
-Note that messages between actors are not signed this is because messages between actors are always spawned by a message from a user -- which are singed by that user. 
+## Dependencies
 
-Filecoin specific reliance
+- Elliptic curve implementations
+  - ECDSA with secp256k1 (BitCoin-style elliptic curve)
```

Comment:
Would be good to have some reference link for this.

Reply:
Will add links to standards